src/RobotsTxt.php

```diff
@@ -50,6 +50,7 @@ public static function readFrom(string $source): self
     public function __construct(string $content)
     {
         $this->disallowsPerUserAgent = $this->getDisallowsPerUserAgent($content);
+        $this->allowsPerUserAgent = $this->getAllowsPerUserAgent($content);
```

Comment:
looks like a dynamic attribute being created here?

Reply:
Deprecated: Creation of dynamic property Spatie\Robots\RobotsTxt::$allowsPerUserAgent is deprecated in /srv/..../vendor/spatie/robots-txt/src/RobotsTxt.php on line 53